msys2-runtime/0019-strace.cc-Don-t-set-MSYS-noglob.patch

```diff
@@ -50,12 +50,17 @@ index 0c8d75e..f7c98bd 100644
 +  See quoted () comment:
 +  "This must have been run from a Windows shell, so preserve
 +     quotes for globify to play with later."
-+
+ 
    const char *cygwin_env = getenv ("MSYS");
    const char *space;
-   if (cygwin_env)
-@@ -360,6 +378,7 @@ create_child (char **argv)
-   char *newenv = (char *) malloc (sizeof ("MSYS=noglob") + strlen (space) + strlen (cygwin_env));
+ 
+-  if (cygwin_env && strlen (cygwin_env) <= 256) /* sanity check */
++  if (cygwin_env && strlen (cygwin_env) <= 256) // sanity check
+     space = " ";
+   else
+     space = cygwin_env = "";
+@@ -364,6 +381,7 @@ create_child (char **argv)
+ 				  + strlen (space) + strlen (cygwin_env));
    sprintf (newenv, "MSYS=noglob%s%s", space, cygwin_env);
    _putenv (newenv);
 +*/
```

Comment:
Because it was there before.

Please note that the `+` you see there is the `+` in `0019-*.patch`, it is *not* an indicator of an line added by *this* commit.

There is a space before the `+`, and *that space* indicates that this line has not been modified in the `.patch` file.

I agree that it is difficult to review diffs of diffs, and I really wish that `msys2-runtime` would not have moved to patches, but rather to [merging rebases](https://github.com/git-for-windows/build-extra/blob/746321513a062a3b8b3a06bd588273ac8f8f3abe/shears.sh#L12-L18).